R/prep_data.R

```diff
@@ -59,10 +59,18 @@
 #'   into a new category, `other`. If numeric, must be in {0, 1}, and is the
 #'   proportion of observations below which levels will be grouped into other.
 #'   See `recipes::step_other`.
+#' @param PCA Integer or Logical. PCA is done to reduce training time,
+#'   particularly for wide datasets. If integer, represents the number of
+#'   principalcomponents to convert the numeric data into. If TRUE, will convert
+#'   numeric data into 5 principal components. Data must also be centered and
+#'   scaled (default).  If 0 or FALSE, the PCA step will be skipped. Default is
+#'   FALSE.
```

Comment:
I got a little lost in this doc string. I'd say, "PCA requires that data is centered and scaled and will set those params to TRUE. Default is FALSE."